### PR TITLE
Update rubocop configuration

### PR DESCRIPTION
**Pull request by backus**

### CODE REVIEW

**Comment by backus** [comment]
@mbj @dkubb this should be ready for review. I might PR more rubocop fixes over the next few days but I'm switching focus to writing a custom cop now

File: Rakefile [hunk 1]
```diff
@@ -28,10 +28,9 @@ desc 'Generate mutation operator listing'
 task :list do
```
Comment (mbj) [inline Rakefile]: Lets kill this task, its broken.

Comment (backus) [inline Rakefile]: Well my changes fixed it I think try running it.

Comment (backus) [inline Rakefile]: Killed

**Comment by mbj** [comment]
@backus Diff wise I'm fine, I'll merge on green build.

File: .rubocop.yml [hunk 2]
```diff
@@ -1,11 +1,7 @@
 AllCops:
-  Include:
-    - 'Gemfile'
```
Comment (dkubb) [inline .rubocop.yml]: Is the `Gemfile` automatically included, or do we want it not to be checked for some reason?

Comment (mbj) [inline .rubocop.yml]: I assumed `Gemfile` is now automatically included.

Comment (backus) [inline .rubocop.yml]: Yeah it is see the commit message. Also: https://github.com/bbatsov/rubocop/blob/master/config/default.yml#L9-L27

**Pull request merged**